pkg/azure/ocpgwdeployer.go

```diff
@@ -436,3 +446,30 @@ func (d *ocpGatewayDeployer) getClients(status reporter.Interface) (
 
 	return nsgClient, nwClient, pubIPClient, nil
 }
+
+func (d *ocpGatewayDeployer) getWorkerNodeImage() (string, error) {
```

Comment:
This could be defined in `ocp/machinesets.go`, which would avoid exposing `ClientForMsd()`.

Reply:
Done.